tests/src/controller/checkout.controller.test.ts

```diff
@@ -331,6 +331,56 @@ describe('Checkout controller', () => {
       expect(responseBody).toStrictEqual(mockedPaymentIntent);
     });
 
+    it('when the user want to pay a one time plan using crypto currencies, then an invoice is created and the specific payload containing the QR Link is returned', async () => {
```

Comment:
I miss a test to ensure that when the interval is not lifetime, the system handles it. We would not like to find someone paying a sub with crypto hahaha